test/src/unit-udt.cpp

```diff
@@ -809,6 +809,10 @@ TEST_CASE("an incomplete type does not trigger a compiler error in non-evaluated
     static_assert(!is_constructible_patched<json, incomplete>::value, "");
 }
 
+// the code below warns about t in MSVC 2015 - this could be a bug
+DOCTEST_MSVC_SUPPRESS_WARNING_PUSH
```

Comment:
Same class of error as the other one, `sizeof(t)` is a compile-time constant based only on `T`, not on the value of `t`.

Reply:
Thanks, I also fixed it.